### ISSUE TITLE
Panic error messages returned by DBSP runtime are not deterministic

### ISSUE THREAD

**Issue by mihaibudiu** [feldera/feldera#1232]
#1215 captures panic errors from multiple worker threads in a common data structure. The goal is to capture the root cause of a panic that kills the runtime. However, this capture seems to be non-deterministic.

This can be reproduced by using the code from #1213 and running a few times the following command:
```
mvn test -Dtest=PostgresInt8Tests -DfailIfNoTests=false
```

This will pass or fail intermittently.

**Comment by mihaibudiu** [comment]
I hope that this is fixable... these threaded things are always tricky.
I could not reproduce this with a single test, perhaps it has something to do with how threads are assigned different tests?
This is how a failing test is described:
```
#[test]
#[should_panic(expected = "ParseIntError")]
pub fn test55() {
```

**Comment by ryzhyk** [comment]
Should be resolved in #1213
